joomla/joomla-cms
# Regression user notes in modal layout

#### Summary of Changes

This PR solves a regression in users notes modal view.
###### Before PR

![before-pr](https://cloud.githubusercontent.com/assets/9630530/13338017/c0450d18-dc16-11e5-97d9-10a67b38ef74.png)
###### After PR

![after-pr](https://cloud.githubusercontent.com/assets/9630530/13338026/c3d2de38-dc16-11e5-9d64-44f9a5372d61.png)
#### Testing Instructions
1. Use latest staging
2. Create some notes for a user.
3. Go to users list and click in show user notes.
4. You will see the problem described in image above.
5. Apply this patch
6. Repeat 3. and you'll see the correct notes.
